Guide_for_Contributors_New_to_Github.md

```diff
@@ -0,0 +1,124 @@
+1. Select ‘Create a new branch for this commit’
+1. Create a name for the new branch that explains **what it is** 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Edit2.png" />. 
+1. Click Propose Changes. 
+1. Click on the Pull Requests tab. Do NOT click the green ‘Create Pull Request’ button that appears at the bottom of the next screen.
+
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/Edit7.png" />
+
+## 4. Submit a Pull Request (Submit edits for review)
+
+1. Click ‘New Pull Request’. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR1.png" />
+1. Make sure that your base repository is vmware-tanzu/tanzu-dev-po..., and the head repository is your account.  
+
+1. Change the dropdown for your “compare: main” to the name of the branch you have created. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR2.png" />
+1. Scroll down to check that text highlighted in green is the change you want to make.   
+1. Click Create Pull Request
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR5.png" />
+1. Click Create Pull Request again. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR6.png" />
+1. If you scroll down, you can see that the edits are going through some automated checks. 
+    <img style="float: left;" src="https://github.com/Weimankow/tanzu-dev-portal/blob/Guide-for-Contributors-New-to-Github/static/images/guides/guide-for-contributors-new-to-github/PR7.png" />
+1. If you want to, you can keep the screen running until all the tests have passed, just to have a peace of mind. In the list of checks, some people can see a row below WIP called netlify. If you see it, you can click on the netlify row after all the checks have completed to preview how it will look on the site.
+1. Otherwise, feel free to close GitHub and continue with your life. The checks will continue running even if you leave the page. Once the change passes through all the checks, it will land on the reviewers’s plate, and all you have to do is wait for it to be reviewed, approved and merged. 
+
+1. You’ve successfully suggested a change to the practices site! 
+
+
+> ## Short FAQ
```

Comment:
One thing that might trip contributors up is if the spell checker fails due to a term it doesn't know about.

We could add some content here describing how to update the dictionary. Alternatively, we can just leave that out to keep this guide from becoming too big a hurdle, and instead have a maintainer reach out if someone gets stuck on that step.

Reply:
Well.. the first time I submitted a PR to Tanzu dev portal, the spell check called out `wireframe` as a spelling mistake. I went back to change it to `wire frame` so that it'll pass the test. 

I'm not sure how hard it is to update the dictionary - but perhaps pointing contributors to contact the maintainer (I'm guessing that's someone who can help with updating the dictionary) might be easier.

Reply:
Can we add that later? There is already a ton of content here. We can make multiple small additions after this is merged.